posix/subsystem/src/tmp_fs.cpp

```
@@ -316,6 +320,15 @@ struct DirectoryNode final : Node, std::enable_shared_from_this<DirectoryNode> {
 
 	async::result<frg::expected<Error, std::shared_ptr<FsLink>>> mksocket(std::string name) override;
 
+	async::result<frg::expected<Error>> rmdir(std::string name) override {
+		auto result = co_await unlink(name);
+		if(!result) {
+			std::cout << "tmpfs: Unexpected failure from unlink()" << std::endl;
+			co_return {};
```

Shouldn't we return an error here?